src/app/components/publiclists/teamlist/teamlist.component.html

```diff
@@ -1,91 +1,63 @@
-          <div *ngIf="isOnChallengePage && !isParticipated" class="participants-team-list">
-            <div *ngIf="!paginationDetails.showPagination" class="content">{{paginationDetails.paginationMessage}}</div>
-            <ul class="rm-pad">
-              <li *ngFor="let value of allTeams">
-                <input [(ngModel)]="teamId" type="radio" name="selectExistTeam" class="with-gap selectTeam"
-                  id="{{value.id}}" value="{{value.id}}" (click)="selectedParticipantTeam(value)">
-                <label for="{{value.id}}"></label>
-                <div *ngIf="value.team_url != ''" class="radio-button-text pointer pointer content">
-                  <strong class="text-med-black fw-semibold">Team: </strong>
-                  <a class="orange-text" href="{{value.team_url}}" target="_blank">{{value.team_name}}</a>
-                  <br><strong class="text-med-black fw-semibold">Created By: </strong>{{value.created_by}}
-                </div>
-                <div *ngIf="value.team_url == ''" class="radio-button-text pointer pointer content">
-                  <strong class="text-med-black fw-semibold">Team: </strong>{{value.team_name}}
-                  <br><strong class="text-med-black fw-semibold">Created By: </strong>{{value.created_by}}
-                </div>
-                <div class="clearfix"></div>
-              </li>
-
-              <div class="pagination" *ngIf="paginationDetails.showPagination">
-                <div class="row row-lr-margin">
-                  <div class="col-md-6 col-sm-12 left-align col-lr-pad">
-                    <button [ngClass]="paginationDetails.isPrev" class="btn-floating btn-pagination waves-effect waves-light " 
-                    (click)="loadPaginationData(previous)">
-                      <i class="fa fa-chevron-left"></i>
-                    </button>
-                    <span class="pagination-title">
-                      <strong class="text-med-black fw-semibold content">
-                        Page {{paginationDetails.currentPage}} of {{paginationDetails.totalPage}}
-                      </strong></span>
-                    <button [ngClass]="paginationDetails.isNext" class="btn-floating btn-pagination waves-effect waves-light" 
-                    (click)="loadPaginationData(next)">
-                      <i class="fa fa-chevron-right"></i>
-                    </button>
-                  </div>
-                  <div class="col-md-6 col-sm-12 col-lr-pad">
-                    <div class="create-challenge align-right" *ngIf="isOnChallengePage">
-                      <button [disabled]="!selectedTeam" class="btn btn-waves-effect grad-rec-btn waves-dark ev-btn-dark 
-                        grad-btn-dark fw-light fs-14" (click)="participateInChallenge()">
-                        Continue
-                      </button>
-                    </div>
-                  </div>
-                </div>
+<section class="ev-sm-container ev-view ">
```

Comment:
Minor: When I create a team having the name same as one of the already existing team, it does not create the team, nor does it throws any error toast. Maybe an appropriate error would make sense here.

Reply:
Done